test/index.test.js

```diff
@@ -529,5 +529,93 @@ Previous tag: ''
         expect.assertions(1)
       })
     })
+
+    it('when only using major.minor version template', async () => {
```

Comment:
Can you create a new `describe` block called something like "with version-template config"?